src/golang/cmd/server/handler/archive_notification.go

```diff
@@ -17,8 +17,10 @@ import (
 // Method: POST
 // Params: notificationId
 // Request:
+//
```

Comment:
Seems like a bunch of these files weren't previously linted? Or linter rules changed somehow?

Reply:
I think the golang linter version got updated in our Git action, and the updated version caused these changes. I was originally using the old version which didn't pass the linter test.